### PR TITLE
chore(ai): add keyboard navigation for conversation

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- Added keyboard navigation to mimic Facebook Messenger and other chat applications
- Tabbing through the component only hits the most recent message and it's action buttons
- Once focused on a message, arrow navigation up and down goes to other messages
- Also set autofocus to the input area

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by thaddmt** [comment]
focus

**Review by esauerbo — DISMISSED** [review]
LGTM!

**Review by timngyn — DISMISSED** [review]
makes sense to me

File: packages/react-ai/src/ai-conversation-styles.css [hunk 1]
```diff
@@ -166,6 +166,7 @@
   align-self: stretch;
   overflow: hidden;
   resize: none;
+  outline: none;
```
Comment (dbanksdesign) [inline packages/react-ai/src/ai-conversation-styles.css:169]: Just leaving a note here for posterity: we probably shouldn't remove the outline for a11y on these default elements?

File: packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx [hunk 2]
```diff
@@ -90,6 +90,14 @@ const TextInput: typeof TextAreaBase = React.forwardRef(
...
     });
 
+    React.useEffect(() => {
+      const textarea = document.getElementById(`text-input`);
+
+      if (textarea) {
+        textarea.focus();
+      }
+    }, []);
+
```
Comment (dbanksdesign) [inline packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx:100]: Is this just mock code that will be replaced later? Seems a bit weird to be finding an element in the document by an ID..

**Review by dbanksdesign — COMMENTED** [review]
Just leaving some comments

File: packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx (hunk 2 above)
Comment (thaddmt) [reply packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx:100]: Oh yeah can update this one, was mostly focused on the keyboard navigation stuff

File: packages/react-ai/src/ai-conversation-styles.css (hunk 1 above)
Comment (thaddmt) [reply packages/react-ai/src/ai-conversation-styles.css]: makes sense, fixed

File: packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx (hunk 2 above)
Comment (thaddmt) [reply packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx]: Fixed, thanks for catching!

**Review by dbanksdesign — APPROVED** [review]

**Review by timngyn — APPROVED** [review]

**Pull request merged**